### PR TITLE
Make archon and bigzombie larger, and animate deaths of important things

### PR DESCRIPTION
**Pull request by axc**
Resolves https://github.com/battlecode/battlecode-client/issues/85

Now deaths of neutrals, zombie dens, and archons now have a ring around them.

Some weird changes, so I've made comments below explaining my logic.

### CODE REVIEW

File: src/main/battlecode/client/viewer/render/DrawHUD.java [hunk 1]
```diff
@@ -156,8 +156,8 @@ public void drawRobot(Graphics2D g2, DrawObject r, double size, double
                 g2.drawImage(underImg, null, null);
             }
             g2.setTransform(pushed2);
-            if (r != null && r.isAlive()) {
-                r.drawImmediate(g2, false, false);
+            if (r != null && r.getHealth() > 0) {
```
Comment (axc) [inline src/main/battlecode/client/viewer/render/DrawHUD.java]: r.isAlive() returns true during the death animation, but we don't really want the death animation to show on the HUD. Thus, we'll only draw the robot if it's actually alive.

File: src/main/battlecode/client/viewer/render/DrawHUD.java [hunk 2]
```diff
@@ -156,8 +156,8 @@ public void drawRobot(Graphics2D g2, DrawObject r, double size, double
...
+                r.drawImmediate(g2, false, true, false, false);
```
Comment (axc) [inline src/main/battlecode/client/viewer/render/DrawHUD.java]: Passes the "isHUD = true" argument into drawImmediate

**Comment by kazimuth** [comment]
Those comments would be better as, well, comments.

**Comment by axc** [comment]
Updated

**Pull request merged**